index.js

```diff
@@ -88,6 +88,10 @@ is.empty = function (value) {
     return true;
   }
 
+  if (is.nil(value)) {
```

Comment:
I wonder if this would be simpler as `return !!value` which covers `null`, `undefined`, and `NaN`?